Locale/fr_FR/translations.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 return array(
+    'Mailgun Unknown ID mapping' => 'Mappage d\'ID inconnu de Mailgun',
```

Comment:
I will update the French translation, once it get merged. Don't worry about that.

Reply:
I removed this file so you can add the translation